packages/staking/src/features/store/delegationPortfolioStore/useSyncDelegationPortfolioStore.ts

```diff
@@ -12,9 +12,7 @@ export const useSyncDelegationPortfolioStore = () => {
   const delegationPortfolio = useObservable(walletStoreInMemoryWallet.delegation.portfolio$);
 
   useEffect(() => {
-    if (
-      [delegationDistribution, delegationRewardsHistory, currentEpoch, currentChain, delegationPortfolio].every(Boolean)
-    ) {
+    if ([delegationDistribution, delegationRewardsHistory, currentEpoch, currentChain].every(Boolean)) {
```

Comment:
Thank you for finding a solution! The main reason it appeared is that the code doesn't look clear in intentions here. If `delegationPortfolio` can be `null` can we please change the implementation to be 100% explicit and check for `undefined`?

```suggestion
    if ([delegationDistribution, delegationRewardsHistory, currentEpoch, currentChain, delegationPortfolio].some(isUndefined)) return;
```